### PR TITLE
[16.01] Fix optional watchdog handling.

### PR DESCRIPTION
**Pull request by jmchilton**
Broken with 4086dd61dafe562ec01a8b1fce687a539aedbcfe.

Thanks for bug report from @nsoranzo.

### CODE REVIEW

File: lib/galaxy/tools/toolbox/watcher.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
     from watchdog.observers.polling import PollingObserver
     can_watch = True
 except ImportError:
+    Observer = None
     FileSystemEventHandler = object
     PollingObserver = object
```
Comment (nsoranzo) [inline lib/galaxy/tools/toolbox/watcher.py]: I think also `PollingObserver` can be set to `None`.

**Comment by nsoranzo** [comment]
Works for me, thanks!

**Comment by jmchilton** [comment]
@nsoranzo - good call on PollingObserver, I've made the suggested change.

**Comment by nsoranzo** [comment]
:+1:

**Pull request merged**